src/main/java/expresslibrary/model/person/Person.java

```diff
@@ -102,7 +102,8 @@ public boolean isSamePerson(Person otherPerson) {
         }
 
         return otherPerson != null
-                && otherPerson.getName().equals(getName());
+                && otherPerson.getEmail().equals(getEmail())
+                && otherPerson.getPhone().equals(getPhone());
```

Comment:
Shouldnt this be 
```suggestion
                && (otherPerson.getEmail().equals(getEmail())
                || otherPerson.getPhone().equals(getPhone()));
```

because if you use && that means that two people would be considered the same if they have the same email but different phone number?